src/core/Pills/BasicPills/BasicPills.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useStyles } from './style';
+
+type Variant = 'default' | 'selected';
+
+interface PillsProps {
+  variant: Variant;
+}
+
+const Pills: React.FC<PillsProps> = ({ variant, children }) => {
+  const classes = useStyles();
+  const getVariant = (variant: Variant) => {
+    switch (variant) {
+      case 'selected':
+        return classes.active;
+      default:
+        return '';
+    }
+  };
+  return (
+    <div className={`${classes.root} ${getVariant(variant)}`}>{children}</div>
```

Comment:
now we need to change the div to a Chip...as we extended the base props from chip

Reply:
https://material-ui.com/components/chips/#chip
take a look to see the options available and change acc to need.
Omit variant and colors from the interface as we can give our own variant